### ISSUE TITLE
A question about critic-loss in discrete sac？

### ISSUE THREAD

**Issue by outshine-J** [p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch#85]
I applied the code of discrete sac to a custom discrete action environment. During the training process, I found that the loss of critic did not decrease but increased, and the critic-loss value after the increase was very large, even reaching 200+, what is the problem? Caused, how can I fix it? thanks.

**Comment by fry404006308** [comment]
您好，我是范仁义，您的邮件我已经收到，我会尽快处理，谢谢。

**Comment by outshine-J** [comment]
Added, the same happens even if I crop the reward.

**Comment by Mengyu-Messic** [comment]
@outshine-J 
Hello, I have encountered the same problem, have you solved it?

**Comment by fry404006308** [comment]
您好，我是范仁义，您的邮件我已经收到，我会尽快处理，谢谢。

**Comment by outshine-J** [comment]
@Mengyu-Messic 
You can find the answer by following the link. [](https://github.com/toshikwa/sac-discrete.pytorch/issues/12#issue-708665275). Other than that you can change this by setting a fixed temperature.
